client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -191,19 +195,30 @@ const InstalledInOrPricing = ( {
 	const translate = useTranslate();
 
 	if ( ( sitesWithPlugin && sitesWithPlugin.length > 0 ) || isWpcomPreinstalled ) {
+		/* eslint-disable wpcalypso/jsx-gridicon-size */
+		const isActive = !! activePlugins.find( ( activePlugin ) => activePlugin.slug === plugin.slug );
+
 		return (
-			/* eslint-disable wpcalypso/jsx-gridicon-size */
-			<div className="plugins-browser-item__installed">
-				<Gridicon icon="checkmark" size={ 14 } />
-				{ isWpcomPreinstalled || currentSites.length === 1
-					? translate( 'Installed' )
-					: translate( 'Installed on %d site', 'Installed on %d sites', {
-							args: [ sitesWithPlugin.length ],
-							count: sitesWithPlugin.length,
-					  } ) }
+			<div className="plugins-browser-item__installed-and-active-container">
+				<div className="plugins-browser-item__installed">
+					<Gridicon icon="checkmark" size={ 14 } />
+					{ isWpcomPreinstalled || currentSites.length === 1
+						? translate( 'Installed' )
+						: translate( 'Installed on %d site', 'Installed on %d sites', {
+								args: [ sitesWithPlugin.length ],
+								count: sitesWithPlugin.length,
+						  } ) }
+				</div>
+				{ currentSites.length === 1 && (
```

Comment:
Do we need to check that we have `currentSites` here, or is it checked somewhere else?
```suggestion
				{ currentSites?.length === 1 && (
```